codegen/templates/grpc_client.tmpl

```diff
@@ -78,7 +85,11 @@ func {{$exportName}}(deps *module.Dependencies) Client {
 	if !circuitBreakerDisabled {
 		for _, methodName := range methodNames {
 			circuitBreakerName := "{{$clientID}}"  + "-" + methodName
-			configureCircuitBreaker(deps, timeoutInMS, circuitBreakerName)
+			qpsLevel := ""
```

Comment:
Should this be "default"? That we will never send "" to configureCircuitBreaker which might break things